roles/rabbit-to-bd/handlers/main.yaml

```diff
@@ -0,0 +1,6 @@
+---
+- name: start rabbit-to-db.service
+  service:
+    name: rabbit-to-db.service
+    state: started
```

Comment:
I believe you mean `restarted` here, don't you?

Reply:
yes